mock-thunderhead/src/main/resources/application.yml

```diff
@@ -50,6 +50,7 @@ auth:
       multiaz.enable: true
       select.instance.type.enable: true
       zdu.osupgrade.enable: true
+      backup.restore.permission.checks.enabled: false
```

Comment:
this should be true for local setting

Reply:
we should fix the default policies used for e2e before enabling it. Otherwise, all the tests will fail.
What do you say?

Reply:
Because it is not only for the mock tests, if it is switched off nobody will try it in local dev mode, so you will have less opportunity to uncover potential issues.

Reply:
this is still false, and @bbihari you have approved the pr. Is this request item still necessary or not?

Reply:
@bbihari  My understanding is that the functionality added is not triggered when the cloud provider is set to MOCK.
Do you agree?